### PR TITLE
feat(lft-page): add september talks

### PR DESCRIPTION
**Pull request by christian-vdz**

### CODE REVIEW

File: _posts/2022-09-30-bullet-journal.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (Slashgear) [inline _posts/2022-09-30-bullet-journal.md:1]: **nitpick:** Can't see this conference on this page https://pr-386.dsvmt7xpjktgx.amplifyapp.com/lft/

File: _data/authors.yml [hunk 2]
```diff
@@ -137,3 +137,17 @@ m_schneider:
...
+f_dumas:
+  name: Fabien Dumas
+c_cousin:
+  name: Camille Cousin
+t_courtoison:
+  name: Thibaud Courtoison
+t_sontag:
+  name: Thomas Sontag
+m_agack:
+  name: Moustapha Agack
```
Comment (Slashgear) [inline _data/authors.yml:151]: **praise:** 
```suggestion
```
Replay is not available ?

File: _posts/2022-09-30-bullet-journal.md (hunk 1 above)
Comment (christian-vdz) [reply _posts/2022-09-30-bullet-journal.md:1]: I can see it, maybe refresh the page?
image

**Review by Oliboy50 — APPROVED** [review]

File: _data/authors.yml (hunk 2 above)
Comment (christian-vdz) [reply _data/authors.yml:151]: The replay is available

Comment (christian-vdz) [reply _data/authors.yml:151]: The video is restricted and unavailable in some countries, France is not one of these countries so we are good :D

**Pull request merged**